min_target_height : 0.0
max_target_height : 3.0
num_tracking_frames : 3
matching_distance_threashold : 1.0